lib/ofx/parser/ofx220.rb

```diff
@@ -16,9 +16,14 @@ def build_account_info(node)
           description:       node.search('desc').inner_text.upcase,
           bank_id:           nested_account_info.search('bankid').inner_text,
           id:                nested_account_info.search('acctid').inner_text,
-          type:              ACCOUNT_TYPES[node.search('accttype').inner_text.to_s.upcase],
+          type:              fetch_account_type(node),
         })
       end
+
+      def fetch_account_type(node)
```

Comment:
As mentioned on #1 I believe this is redudant.

Reply:
Its needed as the implementation is slightly different